src/chocolatey/GetChocolatey.cs

```diff
@@ -222,6 +222,15 @@ public IEnumerable<T> List<T>()
             var runner = new GenericRunner();
             return runner.list<T>(configuration, _container, isConsole: false, parseArgs: null);
         }
+
+        public int Count()

```

Comment:
Hrmm. Normally I'd be against it, but I list ListCount works. Or rather `GetListCount()`